src/components/ArticleContent.jsx

```diff
@@ -324,10 +324,11 @@ class ArticleContent extends React.PureComponent {
 
         <div id="details" hidden={!is_expanded} style={{ paddingBottom: '0.75rem' }}>
           <Typography className={classes.abstract} variant="body2">
-            { is_article_page ?
-            <span>{article.abstract}</span>
-                :
-                  <TruncatedText text={article.abstract} maxLength={540} fontSize={12} />
+            { (is_article_page) 
```

Comment:
this looks good logically, but please improve the formatting/style, following @mickaobrien style, to improve readability: for example:
```
              { (is_article_page) ?
              <span>{article.abstract}</span>
                  :
                     (article.abstract == null || article.abstract.length == 0) ?
                     null
                        :
                            <TruncatedText text={article.abstract} maxLength={540} fontSize={12} />
```
Notice the consistent location of  the "?" and ":" operators

Reply:
i found these ["chained ternaries" in the one line](https://medium.com/javascript-scene/nested-ternaries-are-great-361bddd0f340) appealing because they're concise. but, i agree, your formatting is more readable and consistent with the whole code. i'm changing this